### PR TITLE
Improvement to MessageEmbed static value docs

### PR DESCRIPTION
**Pull request by Sanduhr32**
This PR should improve the docs for some constant fields so that butler bots can display the value.

### CODE REVIEW

**Comment by kantenkugel** [comment]
You can use something like `{@value}` iirc... that way it isn't hardcoded in the docs and gets updated automatically once we change the value

**Comment by Sanduhr32** [comment]
will change!

**Comment by kantenkugel** [comment]
Would also be nice if you could explain a bit what each field is instead of just saying its name (what is value for example?)

**Comment by kantenkugel** [comment]
Did you test building javadocs with latest changes? is that the correct usage of `@value`?

**Comment by Sanduhr32** [comment]
👀 tell me how so i do it next time before i try

**Comment by kantenkugel** [comment]
https://i.imgur.com/7ZX56zQ.png

You can just run `gradlew build` in the console, that will do everything (and build jars)

**Comment by kantenkugel** [comment]
You don't have to explain for babies btw^^ "The title is the line under the author". Just a few clarifications because "VALUE" alone doesn't tell much for example. its the value of fields

**Comment by Sanduhr32** [comment]
whups yee forgot the markdown translation from jdocs into hljs in discord

**Comment by kantenkugel** [comment]
you can also just cut out the whole "The character limit of the maximum YYY length is XXXX chacacters." part. Just say something along the lines of "This field denotes the maximum length of a field value. The fields value is XXX" (less robotic)

File: src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java [hunk 1]
```diff
@@ -30,11 +30,37 @@
...
+    /**
+     * The character limit of the maximum text length is {@value} chacacters.<br>
+     * The text equals the description.<br>
+     * {@see https://imgur.com/a/yOb5n}
+     */
     int TEXT_MAX_LENGTH = 2048;
+    /**
+     * The character limit of the maximum url length is {@value} chacacters.<br>
+     * The url is the optional url markdown of the title.<br>
+     * {@see https://imgur.com/a/yOb5n}
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: The `@see` should be separated from the description by one line and not be wrapped in brackets.

File: src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java [hunk 2]
```diff
@@ -30,11 +30,37 @@
  */
 public interface MessageEmbed
 {
+    /**
+     * The character limit of the maximum title length is {@value} chacacters.<br>
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: `<br>` tags at the beginning of the next line

File: src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java [hunk 3]
```diff
@@ -30,11 +30,37 @@
...
+     */
     int URL_MAX_LENGTH = 2000;
+    /**
+     * The character limit of the maximum bot embed length is {@value} chacacters.
+     */
     int EMBED_MAX_LENGTH_BOT = 4000;
+    /**
+     * The character limit of the maximum client embed length is {@value} chacacters.
+     */
     int EMBED_MAX_LENGTH_CLIENT = 2000;
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: These limits need a better description as to what is included etc.

**Comment by MinnDevelopment** [comment]
I want to see an update on this PR

**Comment by MinnDevelopment** [comment]
As there hasn't been an update to this PR after my requested changes I'm closing this.

**Pull request closed without merging**